cmd/trace-agent/remote_config.go

```diff
@@ -112,3 +114,12 @@ func getContainerTags(req *http.Request, cfg *config.AgentConfig, provider api.I
 	}
 	return nil
 }
+
+func normalize(configsRequest *pbgo.ClientGetConfigsRequest) {
+	// err is explicitly ignored as it is not an actual error and the expected normalized service
```

Comment:
Is it worth logging an error at `TRACE` level? Or would it just be super noisy?

Reply:
I don't think it's needed no, err is used to send signals more than actual errors here. The service is normalized anyway.